modules/dedicated-cluster-admin-enable.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// administering_a_cluster/cluster-admin-role.adoc
+
+[id="dedicated-cluster-admin-enable"]
+= Enabling the cluster-admin role to your cluster
+
+The cluster-admin role must be enabled at the cluster level before it can be assigned to a user.
+
+.Procedure
+. In the OpenShift Cluster Manager, select the cluster you want to assign cluster-admin privileges.
+. Under the *Actions* drop down menu, select *Allow cluster-admin access*. 
```

Comment:
Nit - we usually use "dropdown" instead of "drop down", but no firm rule one way or the other that I know of.